src/inspect_ai/model/_providers/_openai_batch.py

```diff
@@ -28,26 +30,33 @@ class CompletedBatchInfo(TypedDict):
     result_uris: list[str]
 
 
-class OpenAIBatcher(Batcher[ChatCompletion, CompletedBatchInfo]):
+ResponseT = TypeVar("ResponseT", bound=BaseModel)
```

Comment:
This generic `TypeVar` is so that we can use this code for either Completions or Responses. It's used to specify the Pydantic model type that represents the result of a batch request.